src/lib.rs

```diff
@@ -45,7 +47,9 @@
     nonstandard_style,
     rust_2018_idioms
 )]
-#![warn(rust_2018_idioms)]
+// END - Embark standard lints v0.3
+// crate-specific exceptions:
+#![allow(clippy::enum_glob_use)]
```

Comment:
Oops, maybe I should have been more clear, you can keep this lint in the lib file, but then allow it specifically in the error module, so that it continues to catch it in the rest of the crate, just not in that one place.

Reply:
Ah, okay. Gotcha!